config/configuration.go

```diff
@@ -25,30 +25,29 @@ const (
 	Table = "table"
 	// KeyColumn is the configuration name of the key column.
 	KeyColumn = "keyColumn"
+
+	testURL   = "test_user/test_pass_123@localhost:1521/db_name"
+	testTable = "test_table"
```

Comment:
please move it to  file configuration_test.go

Reply:
Good point 👍 
Done!